app/assets/stylesheets/_appointments.scss

```diff
@@ -0,0 +1,16 @@
+#new_appointment {
```

Comment:
Avoid using id selectors<br>Selector `new_appointment` should be written in lowercase with hyphens